<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-H2T0JTB" failures="0" tests="6" name="Default test" time="1.033" errors="0" timestamp="2025-02-27T02:21:21 IST">
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="0.279">
    <failure type="java.lang.NoSuchMethodError" message="&amp;apos;boolean io.restassured.common.mapper.resolver.ObjectMapperResolver.isJakartaEEInClassPath()&amp;apos;">
      <![CDATA[java.lang.NoSuchMethodError: 'boolean io.restassured.common.mapper.resolver.ObjectMapperResolver.isJakartaEEInClassPath()'
at io.restassured.config.ObjectMapperConfig.newJakartaEEObjectMapperFactoryOrNullIfNotInClasspath(ObjectMapperConfig.java:265)
at io.restassured.config.ObjectMapperConfig.<init>(ObjectMapperConfig.java:60)
at io.restassured.config.RestAssuredConfig.<init>(RestAssuredConfig.java:41)
at io.restassured.RestAssured.<clinit>(RestAssured.java:422)
at com.fintech.tests.KycVerificationTest.setUp(KycVerificationTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="-1.740603081945E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="-1.740603081949E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="-1.740603081954E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="-1.740603081957E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="@BeforeMethod setUp" time="-1.74060308196E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase classname="com.fintech.mock.WireMockBaseTest" name="@AfterClass tearDownWireMockServer" time="-1.740603081963E9">
    <skipped/>
  </testcase> <!-- @AfterClass tearDownWireMockServer -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testEndToEndKycCheck" time="0.0">
    <skipped/>
  </testcase> <!-- testEndToEndKycCheck -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testInvalidAadhaarFormat" time="0.0">
    <skipped/>
  </testcase> <!-- testInvalidAadhaarFormat -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testInvalidUserScenario" time="0.0">
    <skipped/>
  </testcase> <!-- testInvalidUserScenario -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testKycServiceFailure" time="0.0">
    <skipped/>
  </testcase> <!-- testKycServiceFailure -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testPendingKycStatus" time="0.0">
    <skipped/>
  </testcase> <!-- testPendingKycStatus -->
  <testcase classname="com.fintech.tests.KycVerificationTest" name="testSuccessfulKycCheck" time="0.0">
    <skipped/>
  </testcase> <!-- testSuccessfulKycCheck -->
</testsuite> <!-- Default test -->
